pom.xml

```diff
@@ -206,6 +207,11 @@
       <version>${rmb.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jmockit</groupId>
+      <artifactId>jmockit</artifactId>
+      <version>1.49</version>
+    </dependency>
```

Comment:
I thought this project already included mockito, does jmockit do something different?

Reply:
Hi @marc-johnson , yes, I 've deleted jmockit and reused existing approach.